plugins/websocket_server/WebsocketServer.cc

```diff
@@ -214,6 +227,23 @@ bool WebsocketServer::Load(const tinyxml2::XMLElement *_elem)
   }
   igndbg << "Using port[" << port << "]\n";
 
+  // Get the maximum connection count, if present.
+  elem = _elem->FirstChildElement("max_connections");
+  if (elem)
+  {
+    try
+    {
+      this->maxConnections = std::stoi(elem->GetText());
+    }
+    catch (...)
+    {
+      ignerr << "Failed to convert port[" << elem->GetText() << "] to integer."
+        << std::endl;
+    }
+    igndbg << "Using maximum connetion count of "
```

Comment:
Typo: connetion → connection

Reply:
Fixed in a30524be7c154097aadcd62f4f840657efabf9f6